packages/database/src/index.ts

```
@@ -0,0 +1,9 @@
+import { drizzle } from "drizzle-orm/postgres-js";
+import postgres from "postgres";
+
+async function main() {
+  const client = postgres(process.env.DATABASE_URL!);
+  const db = drizzle({ client });
```

ESLint

[error] 'db' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)